packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1983,7 +1995,9 @@ class ImportedVpc extends VpcBase {
   private readonly cidr?: string | undefined;
 
   constructor(scope: Construct, id: string, props: VpcAttributes, isIncomplete: boolean) {
-    super(scope, id);
+    super(scope, id, {
+      region: props.region ?? undefined,
```

Comment:
This is redundant. `region: props.region` means the same thing as `region: props.region ?? undefined` when `props.region` is potentially undefined.

```suggestion
      region: props.region,
```